src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,48 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPage extends BasePage {
+
+//    WebDriver driver;
+//    WebDriver wait;
+
+    //Locators
+    By emailField = By.cssSelector("[type='email']");
+    By passwordField = By.cssSelector("[type='password']");
+    By submitButton = By.cssSelector("button[type='submit']");
+
+    @FindBy(css = "[test]")
+    WebElement email;
+
+    public LoginPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+//    public LoginPage(WebDriver givenDriver) {
+//        driver = givenDriver;
+//        wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+//    }
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.